### PR TITLE
@orta=> Post artworkReloadNotification when Presentation Mode is toggled

### LINKED ISSUE #127: Fix presentation mode state change for hiding/unhiding artworks
If you turn on presentation mode with unpublished or not for sale works hidden and return to the top view controller, the grid view doesn't properly handle the missing artworks. It shows a blank image for artists who should be completely hidden instead of removing them entirely.

Also, if you turn presentation mode off, the hidden works remain hidden until you move to a display mode of ARTopViewController (Artists/Shows/Albums).

### PR DESCRIPTION
**Pull request by sarahscott**
Fixes #127

### CODE REVIEW

File: Classes/Controllers/Settings/ARLabSettingsMasterViewController.m [hunk 1]
```diff
@@ -99,6 +99,7 @@ - (IBAction)presentationModeButtonPressed:(id)sender
 {
     [self.viewModel togglePresentationMode];
     self.presentationModeToggle.on = [self.viewModel presentationModeOn];
+    [[NSNotificationCenter defaultCenter] postNotificationName:ARUserDidChangeGridFilteringSettingsNotification object:nil];
```
Comment (sarahscott) [inline Classes/Controllers/Settings/ARLabSettingsMasterViewController.m]: I was posting this notification when presentation mode was edited, but not when it was actually activated or deactivated.

**Comment by orta** [comment]
Thank you can write a test, to cover your regression?

**Comment by sarahscott** [comment]
added some tests! hope you're enjoying your adventure to siberia; i imagine it looks something like this:

**Comment by sarahscott** [comment]
lol nvm realized siberia is next year

**Comment by orta** [comment]
10/10 for those tests :+1:

**Comment by orta** [comment]
and yeah, I also wonder if it's also gonna be boring old grass and hills too

**Pull request merged**